common/libs/validators.ts

```diff
@@ -116,9 +116,8 @@ export function isPositiveIntegerOrZero(num: number): boolean {
 // https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki
 // normal path length is 4, ledger is the exception at 3
 export function isValidPath(dPath: string) {
```

Comment:
could use a test. Where is the regex coming from? Handrolled?

Reply:
Hand rolled, I'll provide some tests and some comments to clarify